File: juvenal030/programacao_basica/questao14.py
print(" tempo sem acidentes ")
x = int(input(" informe os dias sem acidentes "))
c = x // 360
z = (x - c *360) // 30
xx = x - c * 360 - z * 30
print(" o tempo sem acidentes eh " + str(c) + " anos " + str(z) + " meses " +
str(xx)+ "dias" )

File: juvenal030/programacao_basica/Questao14.java
import java.util.Scanner;

public class Questao14 {
    public static void main (String [] args ) {
        Scanner s;
        int d;
        int a;
        int m;
        int dd;
        s = new Scanner (System.in);
        System.out.print(" o tempo sem acidentes");
        System.out.print("informe os dias sem acidentes: ");
        d = s.nextInt ();
        a = d / 360;
        m = ( d - a *360) /30;
        dd = d - a *360 - m *30;
        System.out.println(" o tempo sem acidentes" + a + " anos " + m + " meses" + dd + " dias ");
    }
}
